infra-templates/portworx/0/docker-compose.yml

```diff
@@ -23,4 +24,5 @@ portworx:
      - /opt/pwx/bin:/export_bin
      - /var/run/docker.sock:/var/run/docker.sock
      - /var/cores:/var/cores
+     - /proc:/hostproc
```

Comment:
You can't make changes to existing catalog entries as they have already been released to the public.